rails-tasks/app/views/people/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="on_paper">
+
+<h1>cameron kids</h1>
+
+<table>
+  <% @people.each do |person| %>
+    <tr>
+      <td>
+        <a href="/people/<%= person.id %>"><%= person.name %>
+        <% if person.incomplete.length == 0 %>
```

Comment:
Here and in other parts of this file we should prefer to use `count` instead of `length` on our ActiveRecord model classes. This is because AR is designed to detect when you use `count` and it will generate better SQL queries as a result.